### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zax71**
Modified version of the MV Core README.md for MV Inventories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+![GitHub release (with filter)](https://img.shields.io/github/v/release/multiverse/multiverse-inventories)
+[![Discord](https://img.shields.io/discord/325459248047980545?label=discord&logo=discord)](https://discord.gg/NZtfKky)
+[![Support us on Patreon](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.vercel.app%2Fapi%3Fusername%3Ddumptruckman%26type%3Dpatrons&style=flat)](https://patreon.com/dumptruckman)
 
+
+## About
+
+[Multiverse Inveotiries](https://dev.bukkit.org/projects/multiverse-inventories) is an add-on Plugin for [Multiverse core](https://dev.bukkit.org/projects/multiverse-core) that lets players have separate inventories **per world!** This makes it possible to create multi gamemode servers without Velocity or Bungee!
```
Comment (benwoo1110) [inline README.md:17]:
```suggestion
[Multiverse Inventories](https://dev.bukkit.org/projects/multiverse-inventories) is an add-on Plugin for [Multiverse core](https://dev.bukkit.org/projects/multiverse-core) that lets players have separate inventories **per world!** This makes it possible to create multi gamemode servers without Velocity or Bungee!
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,42 @@
-## Multiverse-Inventories
+<p align="center">
+<img src="https://raw.githubusercontent.com/Multiverse/Multiverse-Core/main/config/multiverse2-long.png" alt="Multiverse Logo">
+</p>
 
-Copyright (c) 2011, The Multiverse Team All rights reserved.
```
Comment (benwoo1110) [inline README.md:3]: Make a license file for it

File: README.md [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+[Multiverse Inveotiries](https://dev.bukkit.org/projects/multiverse-inventories) is an add-on Plugin for [Multiverse core](https://dev.bukkit.org/projects/multiverse-core) that lets players have separate inventories **per world!** This makes it possible to create multi gamemode servers without Velocity or Bungee!
+
+Now it's time to create your very own server with Multiverse Inventories, do check out our [Wiki](https://github.com/Multiverse/Multiverse-Core/wiki/Home-(Inventories)) and [Usage Guide](https://github.com/Multiverse/Multiverse-Core/wiki/Basics-(Inventories)) to get started. Feel free to hop onto our [Discord](https://discord.gg/NZtfKky) if you have any questions or just want to have a chat with us!
+
+## Our other amazing sub-modules:
+
+* [Multiverse-NetherPortals](https://github.com/Multiverse/Multiverse-NetherPortals) -> Have separate nether and end worlds for each of your overworlds!
+* [Multiverse-Portals](https://github.com/Multiverse/Multiverse-Portals) -> Make custom portals to go to any destination!
+* [Multiverse-SignPortals](https://github.com/Multiverse/Multiverse-SignPortals) -> Signs as teleporters!
+
```
Comment (benwoo1110) [inline README.md:26]: maybe add a note that mv-core is required

**Review by benwoo1110 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (zax71) [reply README.md:3]: Should I use the same LICENCE as MV Core?

Comment (benwoo1110) [reply README.md:3]: yea, just make sure core is renamed to inventories

**Review by benwoo1110 — APPROVED** [review]

**Pull request merged**